src/router.coffee

```diff
@@ -18,8 +18,8 @@ routes = (
     <Route path='courses/:courseId/tasks/:id/?' name='viewTask' handler={SingleTask} />
     <Route path='courses/:courseId/practice/?' name='viewPractice' handler={SinglePractice} />
     <Route path='courses/:courseId/readings/?' name='taskplans' handler={TeacherTaskPlans} />
-    <Route path='courses/:courseId/readings/new/?' name='createReading' handler={ReadingShell} />
-    <Route path='courses/:courseId/readings/:id/?' name='editReading' handler={ReadingShell} />
+    <Route path='courses/:courseId/:type/new/?' name='createPlan' handler={PlanShell} />
```

Comment:
Yes, I agree.  Do you think a better solution would be to have 4 routes (edit/add reading, edit/add homework)?  And then have the plan shell infer from the url what to do?